deepchecks/vision/checks/distribution/image_property_drift.py

```diff
@@ -123,28 +139,33 @@ def compute(self, context: Context) -> CheckResult:
             value: dictionary containing drift score for each image property.
             display: distribution graph for each image property.
         """
+        assert self._train_properties is not None
+        assert self._test_properties is not None
+
         if sorted(self._train_properties.keys()) != sorted(self._test_properties.keys()):
             raise RuntimeError('Internal Error! Vision check was used improperly.')
 
         # if self.classes_to_display is set, check that it has classes that actually exist
         if self.classes_to_display is not None:
-            if not set(self.classes_to_display).issubset(
-                    map(self._class_to_string, context.train.classes_indices.keys())
-            ):
+            class_to_string = context.train.label_id_to_name
```

Comment:
Seems like this belongs before the assert

Reply:
sorry, not sure if I understood, do you mean that this 'if' statement should be moved before the assert statements?